src/koans/14_recursion.clj

```diff
@@ -3,21 +3,27 @@
 
 (defn is-even? [n]
   (if (= n 0)
-    __
-    (___ (is-even? (dec n)))))
+    true
+    (not (is-even? (dec n)))))
 
 (defn is-even-bigint? [n]
   (loop [n   n
          acc true]
     (if (= n 0)
-      __
+      false
       (recur (dec n) (not acc)))))
 
 (defn recursive-reverse [coll]
-  __)
+  (loop [[head & tail] coll new []]
+    (if-not head
+      new
+      (recur tail (cons head new)))))
 
 (defn factorial [n]
-  __)
+  (loop [no n acc 1]
+    (if (zero? no)
```

Comment:
You could actually stop at 1, because multiplying by one won't change the state.